recaf-ui/src/main/java/me/coley/recaf/ui/control/code/SyntaxArea.java

```diff
@@ -687,6 +692,22 @@ public CompletableFuture<Void> onApplyStyle(int start, List<LanguageStyler.Secti
 		return CompletableFuture.runAsync(() -> setStyleSpans(start, spans), FxThreadUtil.executor());
 	}
 
+	public void addStyle(StringBinding style) {
+		styles.add(style);
+		style.addListener(__ -> reapplyStyles());
```

Comment:
Even when unused we typically give the listener params names. Doing a quick search I've found a few instances of this existing already on `dev3` of which I'll remove.

Reply:
@xxDark ↑